packages/api/src/cache/index.ts

```diff
@@ -185,8 +185,28 @@ export const createCache = (
     return cache(latestCacheKey, () => model.findMany(conditions), rest)
   }
 
+  const deleteCacheKey = async (key: CacheKey) => {
```

Comment:
Perhaps an an alias (or rename) `deleteCacheKey` to `invalidate`?

See: https://github.com/n1ru4l/envelop/blob/8291991e908f1bb5c9fec1ee2e6e2486f191ee2b/packages/plugins/response-cache/src/cache.ts#L27

Reply:
Since both libs called the function `delete` (and `del`) I figured it was best to stay with that nomenclature. I know we talk about "invalidating the cache" but I wonder if that's the proper term to use for what we're doing...invalidation sounds like it's still around, it's just not "valid" any more. Whereas memcached and Redis really seem to delete it from memory and it's no longer available...